### PR TITLE
Add DLL info

### PR DESCRIPTION
**Pull request by dnmokhov**
### Description 
All the DLLs now have info, e.g.,

Screenshot 2023-12-19 002658

Fixes # - _issue number(s) if exists_

- [x] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [x] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [ ] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### CODE REVIEW

File: include/oneapi/tbb/version.h [hunk 1]
```diff
@@ -17,15 +17,21 @@
...
+    #define __TBB_STRING(x) __TBB_STRING_AUX(x)
+#endif
 
 // Product version
 #define TBB_VERSION_MAJOR 2021
 // Update version
 #define TBB_VERSION_MINOR 4
 // "Patch" version for custom releases
-#define TBB_VERSION_PATCH 0
+#define TBB_VERSION_PATCH 1
```
Comment (isaevil) [inline include/oneapi/tbb/version.h:34]: @omalyshe don't we need to update `TBB_INTERFACE_VERSION` to `12041` as well?

Comment (omalyshe) [reply include/oneapi/tbb/version.h:34]: According to the internal discussion we have to update the TBB_INTERFACE_VERSION as well. @dnmokhov Can you please update the PR?

Comment (dnmokhov) [reply include/oneapi/tbb/version.h:34]: Updated.

**Review by omalyshe — APPROVED** [review]
LGTM.

**Pull request merged**